packages/lib/user/service.ts

```diff
@@ -283,3 +285,22 @@ export const userIdRelatedToApiKey = async (apiKey: string) => {
     throw error;
   }
 };
+
+// Function to check if there are any users in the database
+export const getIsFreshInstance = (): Promise<boolean> =>
```

Comment:
This is the user service so all functions should be around getting more information about users. The function `getIsFreshInstance` seems to be in the wrong place. I see two options:
- refactor it to a generic user function `getUserCount()` but this makes it harder to handle every time we would like to check if it's a fresh instance.
- add a new service, e.g. `/lib/instance/service.ts` where we add the function `getIsFreshInstance()`. I think this feels cleaner and is the way to go.